olive/engine/engine.py

```diff
@@ -918,16 +918,18 @@ def _run_pass(
         input_model_number = input_model_id.split("_")[0]
         # Note: the final output model id need contains the accelerator information
         # if the output model is accelerator dependent.
+
+        model_number = self.cache.get_new_model_number()
```

Comment:
if we don't have any models that are cached with models, then this model_number is useless. It would always just return 0 (or maybe 1). 

It was originally added to get an alias for the model without having to refer to the whole model's id in cache